.github/workflows/main.yml

```diff
@@ -49,6 +49,11 @@ jobs:
       uses: codecov/codecov-action@5a1091511ad55cbe89839c7260b706298ca349f7 # v5.5.1
       with:
         token: ${{ secrets.CODECOV_TOKEN }}
+    - name: rewriteDryRun
```

Comment:
this is just an reminder of course you need to choose where to put this best. 

- https://github.com/diffplug/spotless/blob/main/.github/workflows/ci.yml#L34